akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ClientSpec.scala

```diff
@@ -411,6 +412,34 @@ class Http2ClientSpec extends AkkaSpecWithMaterializer("""
       "fail entity stream if advertised content-length doesn't match" in pending
     }
 
+    "support stream for request data" should {
+      abstract class StreamingRequestSent extends TestSetup {
+        val streamId = 0x1
+        val requestStream = TestPublisher.probe[ByteString]()
+        user.emitRequest(HttpRequest(
+          protocol = HttpProtocols.`HTTP/2.0`,
```

Comment:
I don't think setting the protocol is needed

Reply:
Copy pasta from other test where it perhaps makes more sense :) I'll remove.